### DISCUSSION TITLE
Letter et - superscript form

### DISCUSSION THREAD

**Discussion by TheKnightWho** [psb1558/Junicode-font#128] [Ideas]
Hiya - the letter et (ꝫ) seems to have an occasional superscript form - or at least a raised form - when used as a final "us". The 1136 Charter of King Stephen uses it in "testibus":
![image](https://user-images.githubusercontent.com/6895210/159507815-63d6c3e8-9549-4325-afb1-0a0b9b138eb3.png)

Frustratingly, the transcription has what I think is a printing error, and I cannot work out what it's supposed to be:
![image](https://user-images.githubusercontent.com/6895210/159518275-0355bfba-840c-4ecd-8c0a-afb67735a4ba.png)

Para 4.281 of _The elements of abbreviation..._ states that it is "almost always written on the same line as the other letters in the word". While it gives the Lombardic script as an example of when it isn't, which obviously isn't relevant here, it doesn't feel like a stretch to suggest that a word like "testibus" might have warranted special treatment.

Unfortunately that example is the only use of ꝫ in the 1136 charter itself. However, Statutes also includes an engraving of the reverse, which clearly shows it being used on the line in "Libertatibus" (though admittedly in a different hand):
![image](https://user-images.githubusercontent.com/6895210/159515692-1eb20b0a-145e-477f-98d5-79306cd3220b.png)

Would it please be possible to add a superscript glyph for ꝫ?

**Comment by psb1558** [comment]
Could I see more of the context for the charter of K. Stephen? **Edit**: Never mind, I found it. My next question is whether this should be considered a superscript letter or a combining mark, akin to combining -us (U+1DD2)?  To me it looks more like a combining mark than a superscript.

**Reply by jsbien** [reply]
Where exactly the last image comes from? I'm interested in another aspect of the abbreviation of "Libertatibus" and would like to reference it in some future. Moreover, how do you encode the modified letter "b" in Unicode/Junicode?

**Comment by TheKnightWho** [comment]
Sorry for the slow response - I've been caught up with various other things. Having looked at this again, I agree with you that it would make sense to treat this as a combining mark.

**Comment by psb1558** [comment]
This is done. It will be in the next build, on cv84[35].
